ovs-p4rt/ovs_p4rt.cc

```diff
@@ -2596,17 +2627,30 @@ void ConfigIpMacMapTableEntry(struct ip_mac_map_info ip_info,
       ovs_p4rt::GetForwardingPipelineConfig(session.get(), &p4info);
   if (!status.ok()) return;
 
-  if (ip_info.src_ip_addr.ip.v4addr.s_addr != ANY_INADDR &&
-      ip_info.src_ip_addr.ip.v4addr.s_addr != htonl(LOOPBACK_INADDR)) {
+  if (insert_entry) {
+    auto status_or_read_response =
```

Comment:
The purpose of these lines is not immediately obvious. I'd suggest adding a single-line comment:
```c
// Skip insert if source table entry already exists.
```